src/Umbraco.Cms.Api.Management/Services/Signs/HasPendingChangesSignProvider.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Umbraco.Cms.Api.Management.ViewModels;
+using Umbraco.Cms.Api.Management.ViewModels.Document;
+using Umbraco.Cms.Api.Management.ViewModels.Document.Collection;
+using Umbraco.Cms.Api.Management.ViewModels.Tree;
+using Umbraco.Cms.Core;
+
+
+namespace Umbraco.Cms.Api.Management.Services.Signs;
+
+public class HasPendingChangesSignProvider : ISignProvider
```

Comment:
Could you compare with the earlier sign providers that were implemented and add similar XML header comments to the class and methods please?